cocos/2d/CCDrawNode.h

```diff
@@ -312,7 +312,15 @@ class CC_DLL DrawNode : public Node
     virtual void draw(Renderer *renderer, const Mat4 &transform, uint32_t flags) override;
     
     void setLineWidth(int lineWidth);
-    
+
+#ifdef CC_STUDIO_ENABLED_VIEW   // for cocostudio only
+    void csSetLineSmooth(bool enable);
+
+    bool csIsLineSmooth();
+
+    float csGetLineWidth();
```

Comment:
I think it is not a good idea to start with `cs` even it is only be used for cocos studio.